webserver/vhs-sf-perftest/README.md

```diff
@@ -15,7 +15,7 @@ This workflow has four main phases:
 
 - `kubectl` installed and configured with your cluster
 - `kustomize` (v3.1.0+)
-- [StormForge Optimize Controller (`redskyctl`)](https://github.com/thestormforge/optimize-controller) installed on your cluster
+- [StormForge Optimize Controller (`stormforge`)](https://github.com/thestormforge/optimize-controller) installed on your cluster
```

Comment:
Unrelated to this change but the repo doesn't have great installation instructions: maybe we should change this to the docs.